cves/CVE-2016-1656.yml

```
@@ -52,10 +54,13 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: 168f723d0f0ce60d92a6307c754181f6d8644583
-  :note: ''
+  :note: |
+    The fix commit include the code that parse the incoming file name in the
+    http request to be sanitized before passing to the android download controller
 vccs:
-- :commit:
-  :note: 
+- :commit: 99aace34fc2b2ed5f2f62ca4dd61b6d81da9a13d
+  :note: |
+    The commit did not include sanitizer for the http content desposition (filename)
```

```suggestion
    The commit did not include a sanitizer for the http content desposition (filename)
```
desposition? what word do you mean?